tox.ini

```diff
@@ -114,11 +114,17 @@ deps =
     coverage
     coverage-enable-subprocess==1.0
 
+setenv =
+    postgres: SYNAPSE_POSTGRES = 1
+
```

Comment:
I believe this should be magically inherited from the defaults in `testenv`. could you verify one way or the other?